src/main/java/net/slimediamond/espial/bridge/SpongeBridge.java

```diff
@@ -0,0 +1,7 @@
+package net.slimediamond.espial.bridge;
+
+import org.spongepowered.api.event.block.ChangeBlockEvent;
+
+public interface SpongeBridge {
+    Object getRootCause(ChangeBlockEvent event);
```

Comment:
To get the root cause of any event is the same. Consider using just ``Event`` as the parameter

Reply:
Just seen what you use this for. Ignore this comment